sm_project/cl_app/views.py

```diff
@@ -62,6 +63,15 @@ def add_temp_item(request):
             new_item.checklistitem_set.set(item_form.cleaned_data.get('dependencies'))
     return redirect('cl_app:add_checklist')
 
+@login_required(login_url='user_app:login')
+def use_template(request):
+    """Changes the items in the temporary checklist to a template list of items."""
+    temp_checklist = request.user.get_temp_checklist()
+    item_list = ["Introduction", "Hypothesis", "Key Terms", "Literature Review", "Justification", 
```

Comment:
Could this item list that makes up the template be pulled out into a constant variable? Or perhaps even into a variable in the Djanog app settings? Like this it would be easier to modify if this template should be adjusted at some point in the future.